web-common/src/features/dashboards/url-state/convertURLToExplorePreset.ts

```diff
@@ -41,7 +42,7 @@ import {
   V1Operation,
 } from "@rilldata/web-common/runtime-client";
 
-export function convertURLToExplorePreset(
+export async function convertURLToExplorePreset(
```

Comment:
(Apologies if this is outside the scope of this PR) Why do we need this function at all? Shouldn't we always be converting a URL directly to `ExploreState`?

Reply:
I have this on the backlog for a refactor. Out of scope of this PR